src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -25,9 +18,9 @@ public enum DriveMode {
 
     public DriveSubsystem(ChassisMode chassisMode, DriveMode driveMode) {
         if (chassisMode == ChassisMode.B_TEAM) {
-            driveHardware = new DriveSparkMAX(DriveConstants.frontLeftID, DriveConstants.frontRightID, DriveConstants.backLeftID, DriveConstants.backRightID);
+            driveHardware = new DriveSparkMAX(B_Configs.frontLeftID, B_Configs.frontRightID, B_Configs.backLeftID, B_Configs.backRightID);
```

Comment:
Shouldn't all these configs be passed in via the robot configuration class rather than these constants

Reply:
Should the DriveSubsystem constructor have all of the motor controller ids then?